src/main/java/com/restaurant/controller/RatingController.java

```diff
@@ -33,10 +33,10 @@ public class RatingController {
 	 * @param RatingDto
 	 * @return RatingDto {@link com.restaurant.dto.RatingDto}
 	 */
-	@PostMapping("/{id}")
-	public ResponseEntity<RatingDto> createRating(@Valid @RequestBody RatingDto ratingDto, @PathVariable long id) {
+	@PostMapping("/{orderId}/{userId}")
```

Comment:
The path should be meaningful. 
Example:- 
/{orderId}/user/{userId}

Reply:
done